internal/receiver/main.go

```diff
@@ -90,14 +88,13 @@ func (r *Receiver) subscribeForTopic(ctx context.Context, topic string) error {
 		case <-ctx.Done():
 			return nil
 		case msg := <-msgChan:
-
-			//TODO: 1
 			r.log.Info("received message ", msg.UUID)
 			err = r.processMessage(msg)
 			if err != nil {
 				r.log.WithError(err).Error("failed to process message ", msg.UUID)
+			} else {
```

Comment:
It's kind tricky moment, because if message has wrong format, service won't handle it properly and will not acknowledge it and after restart will try to handle it again, so just acknowledge all messages.

Also add validation for action in received messages (line 105)